README.md

```diff
@@ -50,7 +50,8 @@ config files in the docker if needed):
 ```bash
 [sudo] ./p4runtime-sh-docker --grpc-addr <server IP>:<server port> \
   --device-id 0 --election-id 0,1 \
-  --config <path to p4info>,<path to binary config>
+  --config <path to p4info>,<path to binary config> \
+  --role-name <role name for client>
```

Comment:
I suggest omitting this until we have a released version of P4Runtime (v1.4) that supports string roles. This way we can merge this PR now